### DISCUSSION TITLE
Motion detection

### DISCUSSION THREAD

**Discussion by masciotti** [jmcollin78/versatile_thermostat#663] [Q&A] [answered]
When I use the Motion detection feature the icon works regularly so the command is correct, the preset away is added but I don't see any change in temperature or preset when the state changes. Where am I going wrong?

**Comment by jmcollin78** [comment]
Hello, have you read the README ?

Maybe it is due to the delay ? There no preset away for motion but you have to choose the preset 'ECo, Comfort, Boost) when there is a movement or not.

Away is for presence only

**Reply by masciotti** [reply]
![Config](https://github.com/user-attachments/assets/9739be7e-7708-4bc0-8463-7fc279f5945c)
![office_ motion](https://github.com/user-attachments/assets/1b83e917-0be7-4117-b36b-8685f323652a)
![office_no motion](https://github.com/user-attachments/assets/ec36ed0e-1757-481d-b332-d5c7dabccf8c)
Thank you for your reply, I have re-read the README, I have re-entered the FEATURE USE MOTION DETECTION configuration. But as you can see in the attached pictures, the temperature or preset does not change when the sensor is ON or OFF. I waited a long time to see if anything had changed, but nothing.
With the PRESENCE feature, everything works correctly.

**Reply by jmcollin78** [reply, marked as answer]
Hello,

Th description is here: https://github.com/jmcollin78/versatile_thermostat/blob/main/README.md#configure-the-activity-mode-or-motion-detection
I think you miss this sentence: 
![Capture d’écran 2024-11-27 à 21 59 10](https://github.com/user-attachments/assets/def4c499-bb9b-47dc-87cc-0cb9f66a748c)

Your VTherm is not in activity preset. (The running man)
